3-UsingAPIs/Week1/homework/ex4-pokerDiceAll.js

```diff
@@ -20,12 +18,18 @@ const rollDice = require('../../helpers/pokerDiceRoller');
 function rollTheDices() {
   // TODO Refactor this function
   const dices = [1, 2, 3, 4, 5];
-  return rollDice(1);
+  const arrayOfPromises = dices.map(el => rollDice(el))
+
+  Promise.all(arrayOfPromises).then(values => {
+    return Promise.all(values.map(r => r
```

Comment:
There is no need for this second `Promise.all`. Just return the result of the first `Promise.all` (which is a promise itself) and deal with the result in lines 27-28 that you deleted (but shouldn't have).